server/views/formPages/incident/relocationAndInjuries.html

```diff
@@ -288,24 +288,32 @@ <h2 class="govuk-heading-l govuk-!-margin-top-6">Injuries sustained and healthca
                   }}
                   {{ govukRadios({
                     classes: "govuk-radios--inline",
-                    idPrefix: 'staff-member-went-to-hospital[0][name]',
                     name: 'staffMemberWentToHospital[0][name]',
                     fieldset: {
                       legend: {
                         text: "Did they go to hospital?"
                       }
                     },
-    
                     items: [
                       {
                         value: "yes",
                         text: "Yes",
-                        checked: data.staffMemberWentToHospital[0].name === 'yes'
+                        id: 'staff-member-went-to-hospital[0]-yes',
+                        checked: data.staffMemberWentToHospital[0].name === 'yes',
+                        attributes: {
+                          'data-name': 'staffMemberWentToHospital[%index%][name]',
+                          'data-id': 'staff-member-went-to-hospital[%index%]-yes'
```

Comment:
these attributes are required - after the clone, the js will find the attributes and do a find and replace on the name and id using these templates.

Reply:
might need to do something similar in the case where there are some items already persisted.